homeassistant/components/nut/sensor.py

```diff
@@ -920,23 +936,39 @@
         entity_category=EntityCategory.DIAGNOSTIC,
         entity_registry_enabled_default=False,
     ),
-    "ambient.humidity": SensorEntityDescription(
+    "ambient.humidity": NutSensorEntityDescription(
         key="ambient.humidity",
         translation_key="ambient_humidity",
         native_unit_of_measurement=PERCENTAGE,
         device_class=SensorDeviceClass.HUMIDITY,
         state_class=SensorStateClass.MEASUREMENT,
         entity_category=EntityCategory.DIAGNOSTIC,
     ),
-    "ambient.temperature": SensorEntityDescription(
+    "ambient.humidity.status": NutSensorEntityDescription(
+        key="ambient.humidity.status",
+        translation_key="ambient_humidity_status",
+        device_class=SensorDeviceClass.ENUM,
+        options=AMBIENT_THRESHOLD_STATUS_OPTIONS,
+        ambient_threshold_value_fn=lambda x: x.lower() if isinstance(x, str) else None,
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
+    "ambient.temperature": NutSensorEntityDescription(
         key="ambient.temperature",
         translation_key="ambient_temperature",
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
         device_class=SensorDeviceClass.TEMPERATURE,
         state_class=SensorStateClass.MEASUREMENT,
         entity_category=EntityCategory.DIAGNOSTIC,
     ),
-    "watts": SensorEntityDescription(
+    "ambient.temperature.status": NutSensorEntityDescription(
+        key="ambient.temperature.status",
+        translation_key="ambient_temperature_status",
+        device_class=SensorDeviceClass.ENUM,
+        options=AMBIENT_THRESHOLD_STATUS_OPTIONS,
+        ambient_threshold_value_fn=lambda x: x.lower() if isinstance(x, str) else None,
```

Comment:
Could be methodcaller instead of a lambda

Reply:
context: I can't see how this can be anything other than a `str` based on https://github.com/bdraco/aionut/blob/main/src/aionut/client.py

Reply:
If the value is missing than the sensor than available should return `False` and the state is never examined

Reply:
I can update.  I am not familiar with methodcaller though I have now read up on it a bit.  Is there a good example I should pattern match from since the powerwall component is the only one that uses methodcaller and it is a bit different.  Ideally one where the state is set to never examined.